config/initializers/session_store.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.session_store :cookie_store, expire_after: 1.hour
```

Comment:
I think our SSO guidance says [90 minutes](https://devcenter.heroku.com/articles/add-on-single-sign-on#signing-in-the-user-on-redirect). I'm fine with shorter, but we may want to be consistent.

Reply:
k good idea

Reply:
done at 0df627f